aana/deployments/pyannote_speaker_diarization_deployment.py

```diff
@@ -47,7 +47,7 @@ class PyannoteSpeakerDiarizationConfig(BaseModel):
     model_config = ConfigDict(protected_namespaces=(*pydantic_protected_fields,))
 
 
-@serve.deployment
+@serve.deployment(health_check_timeout_s=180)
```

Comment:
Discussion point: As I understand it waits 3 x 180sec = 9 minutes before restarting the deployment. This means diarization inference should be completed at all costs before 9 minutes. I guess your work on processing long videos should address it. if it will cut the long videos, this should be fine.

Reply:
We can keep it 3min for now. I will do a speed benchmark when adding a new diarization model and we can update it if necessary.